lib/SearchAndSort/ConnectedSource/StripesConnectedSource.js

```diff
@@ -74,6 +75,11 @@ export default class StripesConnectedSource {
     // ... and stripes-connect notices this change and does the rest by magic
   }
 
+  // fetch by offset which is passed in as index. _askAmount is ignored.
```

Comment:
If this is a brand new function, why are we writing it with an argument we ignore?

Reply:
Good point. I can remove it.